test/ctia/entity/asset_mapping_test.clj

```diff
@@ -0,0 +1,95 @@
+(ns ctia.entity.asset-mapping-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.test :refer [deftest is are join-fixtures testing use-fixtures]]
+            [ctia.entity.asset-mapping :as asset-mapping]
+            [ctia.test-helpers.aggregate :as aggregate]
+            [ctia.test-helpers.auth :as auth]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.crud :refer [entity-crud-test]]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.field-selection :as field-selection]
+            [ctia.test-helpers.http :as http]
+            [ctia.test-helpers.pagination :as pagination]
+            [ctia.test-helpers.store :as store]
+            [ctim.examples.asset-mappings :refer [new-asset-mapping-minimal
+                                                  new-asset-mapping-maximal]]))
+
+(use-fixtures :once (join-fixtures [mth/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(defn additional-tests [_ asset-mapping-sample]
+  ;; TODO: write one
+  (testing "GET /ctia/asset-mapping/search"
+    ;; only when ES store
+    (when (= "es" (get-in @ctia.properties/properties [:ctia :store :asset-mapping]))
+      (are [term check-fn expected desc] (let [response (helpers/get
+                                                         "ctia/asset-mapping/search"
+                                                         :query-params {"query" term}
+                                                         :headers {"Authorization" "45c1f5e3f05d0"})]
+                                           (is (= 200 (:status response)))
+                                           (is (= expected (check-fn response)) desc))
+
+        "specificity:\"unique\""
+        (fn [r] (-> r :parsed-body first :specificity))
+        (-> asset-mapping-sample :specificity)
+        "Searching Asset Mapping by specificity"
+
+        ;; TODO: Add more cases
+
+        ))))
+
+(deftest asset-mapping-routes-test
+  (store/test-for-each-store
+   (fn []
+     (helpers/set-capabilities! "foouser" ["foogroup"] "user" auth/all-capabilities)
+     (whoami-helpers/set-whoami-response http/api-key "foouser" "foogroup" "user")
+     (entity-crud-test
+      {:entity           "asset-mapping"
+       :example          new-asset-mapping-maximal
+       :invalid-tests?   false
+       :update-tests?    true
+       :search-tests?    false
+       :update-field     :source
+       :additional-tests additional-tests
+       :headers          {:Authorization "45c1f5e3f05d0"}}))))
+
+#_(deftest asset-mapping-pagination-test
```

Comment:
maybe that's an oversight but you forgot to enable those tests

Reply:
Yeah, I was so overly excited that other (default) tests are now passing that I've forgotten to check asset-mapping tests. Previously I got stuck badly, until @ereteog  found the issue and fixed it for me. I will make sure these tests are working as well.